src/nvim/testdir/runnvim.vim

```diff
@@ -43,9 +43,6 @@ function Main()
         \])
   write
   if results[0] != 0
-    if results[0] != -1
-      call jobstop(job)
-    endif
```

Comment:
Actually why not use jobwait(0) to check if the job exists. In case bwipeout behavior changes for some reason

Reply:
Wouldn't `:cquit` kill any jobs anyway?

Reply:
yes

Reply:
ok, good then I think.